src/utils/daos_hdlr.c

```diff
@@ -2948,6 +2963,467 @@ fs_copy_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+static int
+cont_copy_recx_single(daos_key_t *dkey,
```

Comment:
@dsikich Future improvement:
Now that we have these functions here and in mpifileutils, we should consider centralizing them in some way.